rllib/models/tf/attention_net.py

```diff
@@ -551,7 +551,10 @@ def forward(
 
     @override(ModelV2)
     def get_initial_state(self) -> Union[List[np.ndarray], List[TensorType]]:
-        return []
+        return [
```

Comment:
I think this belongs in two places: Here and also with the Keras wrapper further down. Our attention test for example will use the Keras wrapper and this will not make it work I believe.

Reply:
I've fixed this now in https://github.com/ray-project/ray/pull/30626/files/26e519d6df1d958ea92a8863f10a1078346e589f#diff-c6890cd20e9c768a4efaa52dcc9a29a8834fddb81592710c147e63e7bdb7310d